### PR TITLE
add Lighthouse 10 release post

### PR DESCRIPTION
**Pull request by brendankenny**
I'm retaking two screenshots, but otherwise should be good for review!

### CODE REVIEW

File: site/en/blog/lighthouse-10-0/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  - chrome-112
+---
+
+[Lighthouse is a website auditing tool](/docs/lighthouse/overview/) that helps developers with opportunities and diagnostics to improve the user experience of their sites.
+
+Lighthouse 10 is available immediately on the [command line through npm](https://www.npmjs.com/package/lighthouse), in [Chrome Canary](https://www.google.com/chrome/canary/), and in [PageSpeed Insights](https://pagespeed.web.dev/). It will land in Chrome stable in Chrome 112.
+
+## Scoring changes
+
+The venerable Time To Interactive (TTI) metric is being removed in Lighthouse 10, with its 10% score weight shifted to Cumulative Layout Shift (CLS), which will now account for 25% of the [overall performance score](/docs/lighthouse/performance/performance-scoring/#lighthouse-10). Ideally this change will increase focus on CLS for sites that still need fixes for unnecessary layout shifts.
```
Comment (brendankenny) [inline site/en/blog/lighthouse-10-0/index.md:23]: perf score link will work after https://github.com/GoogleChrome/developer.chrome.com/pull/5213 lands

File: site/en/blog/lighthouse-10-0/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+### Paste-preventing inputs
+
+The old Best Practices audit "Allows users to paste into password fields" has been expanded to now check that pasting into any (non-readonly) input field will work. For most sites, preventing pasting is a net-negative user experience and prevents legitimate safety and accessibility workflows.
+
+The new audit is now ["Allows users to paste into input fields" (`paste-preventing-inputs`)](/docs/lighthouse/best-practices/paste-preventing-inputs/).
+
+## Node users
+
+If you use Lighthouse as a Node library, there are a few programmatic breaking changes in this release you may need to account for. See the [10.0 changelog](https://github.com/GoogleChrome/lighthouse/releases/tag/v10.0.0) for full details.
```
Comment (brendankenny) [inline site/en/blog/lighthouse-10-0/index.md:49]: changelog link will work after https://github.com/GoogleChrome/lighthouse/pull/14594 lands and is released

File: site/en/blog/lighthouse-10-0/index.md (hunk 1 above)
Comment (malchata) [inline site/en/blog/lighthouse-10-0/index.md:23]:
```suggestion
The venerable [Time To Interactive (TTI)](https://web.dev/tti/) metric is being removed in Lighthouse 10, with its 10% score weight shifted to [Cumulative Layout Shift (CLS)](https://web.dev/cls/), which will now account for 25% of the [overall performance score](/docs/lighthouse/performance/performance-scoring/#lighthouse-10). Ideally this change will increase focus on CLS for sites that still need fixes for unnecessary layout shifts.
```

**Review by malchata — APPROVED** [review]
LGTM, minus a couple of nits.

**Pull request merged**